notebooks/1_LinearMethods.ipynb

```diff
@@ -1175,18 +1097,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "table_from_dict([mds.statistics(X_train), mds.statistics(X_test)], \n",
-    "                 headers = [\"Training\", \"Testing\"], \n",
-    "                 title=\"MDS\")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## LR\n",
-    "\n",
-    "Unlike PCA, which takes the number of PCA components in the constructor, the linear regression class takes the regularization as a parameter."
+    "from sklearn.linear_model import LinearRegression as LR\n",
```

Comment:
I think it might be unnecessary to import LinearRegregation because we've never used it.

Reply:
This is to show the two sklearn imports

Reply:
Okay.